pkg/manager/impl/node_execution_manager_test.go

```diff
@@ -132,6 +132,9 @@ func TestCreateNodeEvent(t *testing.T) {
 		StartedAt: occurredAtProto,
 		CreatedAt: occurredAtProto,
 		UpdatedAt: occurredAtProto,
+		TargetMetadata: &admin.NodeExecutionClosure_TaskNodeMetadata{
```

Comment:
why is this change necessary? we shouldn't be initializing the metadata in the closure unless it's sourced from the event itself, right?

Reply:
Ah sorry, I didn't realize the event under test actually had a non-empty [TargetMetadata](https://github.com/flyteorg/flyteadmin/blob/6a275357525ae9b6f31a5a62a55095849721cfd4/pkg/manager/impl/node_execution_manager_test.go#L68) - and we should therefore expect to see the closure here have an initialized TargetMetadata